website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md

```diff
@@ -2,13 +2,13 @@
 sidebar_position: 2
 ---
 
-# Whole Genome Germline Single Sample v2.5.0 Methods
```

Comment:
Did we ever figure out a way to keep the old version methods sections in an archive?

Reply:
We added a section to the methods docs that links to the previous methods markdown pages.